### PR TITLE
[Tiny-fix] Add show user stmt impl.

### PR DESCRIPTION
**Pull request by SaintBacchus**
## Problem Summary:
`Show User` stmt always result empty because `ShowExecutor.java` miss the impl of `handleShowUser`

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (Yes)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by xy720 — COMMENTED** [review]
I think we already have `show grants` stmt to show users.

**Comment by xy720** [comment]
My mistake. This stmt is used to show current login user

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowUserStmt.java [hunk 1]
```diff
@@ -28,9 +28,8 @@ public class ShowUserStmt extends ShowStmt {
 
     static {
         ShowResultSetMetaData.Builder builder = ShowResultSetMetaData.builder();
-        for (String title : AuthProcDir.TITLE_NAMES) {
-            builder.addColumn(new Column(title, ScalarType.createVarchar(30)));
-        }
+        builder.addColumn(new Column("User", ScalarType.createVarchar(30)));
```
Comment (xy720) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowUserStmt.java:31]: We should show UserIdentity to user, just like Mysql.
For example, `'root'@'localhost'.`

Comment (SaintBacchus) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowUserStmt.java:31]: OK, it will may be the same as `select user()` stmt.

Comment (xy720) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowUserStmt.java:31]: builder.addColumn(new Column(col, ScalarType.createVarchar(100)));

Comment (SaintBacchus) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowUserStmt.java]: Got

**Comment by jackwener** [comment]
There are `UT` failed. You can click `details` and see the detail by click `as guest`

**Comment by SaintBacchus** [comment]
@jackwener OK, I will rebase it and fix regression

**Comment by SaintBacchus** [comment]
@jackwener UT has been resolved.

**Review by jackwener — DISMISSED** [review]
LGTM

**Review by hf200012 — DISMISSED** [review]
LGTM

File: docs/zh-CN/sql-manual/sql-reference/Show-Statements/SHOW-USER.md [hunk 2]
```diff
@@ -31,9 +31,19 @@ under the License.
 SHOW USER
 
 ### Description
+该语句用于展示当前登录用户的用户名
 
-### Example
+语法：
```
Comment (morningman) [inline docs/zh-CN/sql-manual/sql-reference/Show-Statements/SHOW-USER.md:36]: Add an example result and explain each column in result set.

Comment (SaintBacchus) [reply docs/zh-CN/sql-manual/sql-reference/Show-Statements/SHOW-USER.md:36]: @morningman updated code

**Pull request closed without merging**